docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29deb98b",
+   "metadata": {},
+   "source": [
+    "For the problem encoding we use: \n",
+    "\n",
+    "- Configuration qubits: qubits that are used to describe the configurations and the relative position of the different beads\n",
+    "- Interaction qubits: qubits that encode interacations between the different aminoacids\n",
+    "\n",
+    "For our case we use a tetrahydral lattice (diamond shape lattice) where we encode the movement through the configuration qubits (see image below). \n",
+    "\n",
+    "<img src=\"aux_files/lattice_protein.png\" width=\"300\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f571d4a8",
+   "metadata": {},
+   "source": [
+    "The Hamiltonian of the system for a set of qubits $\\mathbf{q}=\\{\\mathbf{q}_{cf}, \\mathbf{q}_{in}\\}$ is \n",
+    "\n",
+    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}) $$\n",
+    "\n",
+    "where \n",
+    "- $H_{gc}$ is the geometrical constrain term (governing the growth of the primary sequence of aminoacids without bifurcations\n",
+    "- $H_{ch}$ is the chirality constrain (enforcing the righ stereochemistry for the system)\n",
+    "- $H_{in}$ is the interaction energy terms of the system. In our case we consider only nearest neighbor interactions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d99a5f",
+   "metadata": {},
+   "source": [
+    "Further details about the used model and the encoding of the problem can be found in [2].\n",
+    "\n",
+    "<font size='2'>[2] A.Robert, P.Barkoutsos, S.Woerner and I.Tavernelli, Resource-efficient quantum algorithm for protein folding, NPJ Quantum Information, 2021 </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8f541d04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.random_interaction import RandomInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.miyazawa_jernigan_interaction import MiyazawaJerniganInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.peptide.peptide import Peptide\n",
+    "from qiskit_nature.problems.sampling.protein_folding.protein_folding_problem import ProteinFoldingProblem\n",
+    "\n",
+    "from qiskit_nature.problems.sampling.protein_folding.penalty_parameters import PenaltyParameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fced9189",
+   "metadata": {},
+   "source": [
+    "### Protein Main Chain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "350c031c",
+   "metadata": {},
+   "source": [
+    "The Protein consists of a main chain that is a linear chain of aminoacids. For the naming of different residues we use the one-letter code as defined in Reference [3]. Further details about the naming and the type of aminoacids can also be found in [4].\n",
```

Comment:
> aminoacids

Above you wrote "amino acids" as two words. I would stick to that. Maybe do a search-and-replace in case I overlooked more occurrences.